Makefile

```diff
@@ -23,20 +23,14 @@ SHELL=/bin/bash
 .PHONY: all
 all: build
 
-.PHONY: generate
-generate:
-	@if [[ -n $$(git status --porcelain packs/) ]]; then \
-		go generate ./pkg/draft/pack/generated; \
-	fi
-
 .PHONY: build
-build: generate
+build:
```

Comment:
This change removes the prerequisite to run `make generate` before `make build`. Since we have removed the default draft packs and are just adding the default pack repository via `draft pack-repo add`, `make generate` is no longer necessary and thus has been removed.

Reply:
Yeah, but now your Makefile looks like this:

```
.PHONY: build
build:
build:
	GOBIN=$(BINDIR) $(GO) install $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' github.com/Azure/draft/cmd/...

# usage: make clean build-cross dist APP=draft|draftd VERSION=v2.0.0-alpha.3
.PHONY: build-cross
build-cross:
build-cross: LDFLAGS += -extldflags "-static"
build-cross:
```

Reply:
oh derp. Nice catch. Lemme fix that.